nix/distribution.nix

```diff
@@ -26,7 +26,6 @@
     in {
       prismlauncher-unwrapped = prev.qt6Packages.callPackage ./pkg {
```

Comment:
```suggestion
      prismlauncher-unwrapped = prev.callPackage ./pkg {
```
this has been moved to `by-name` in nixpkgs and thus no longer uses anything in `qt6Packages` scope directly

Reply:
Done